common/lib/xmodule/xmodule/x_module.py

```diff
@@ -361,6 +362,14 @@ def xblock_kvs(self):
         self.save()
         return self._field_data._kvs  # pylint: disable=protected-access
 
+    @lazy
+    def _unwrapped_field_data(self):
+        """
+        This property hold the value _field_data here before we wrap it in
+        the LmsFieldData or OverrideFieldData classes.
+        """
+        return self._field_data
```

Comment:
Definitely for the second one. I think the first one is ok, since it's just calling `set_many`, not doing `self._field_data = ...`. And yes, calling invalidate is the relevant thing.